review/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+# flake8: noqa
+from conftest import client, professional, profile1, user1  # noqa
+
+# Only by creating this file I can use `/conftest.py` within `/review/tests/confteset.py`
```

Comment:
Please remove

Reply:
Please refer this
https://docs.pytest.org/en/7.1.x/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files

Reply:
Without this file or its content, `PyTest` initialization will fail, because it fails to import **client, professional, profile1, user1** from `conftest`, because the other option is to import directly from `review/tests/conftest.py`. This does not work for me.

Reply:
There is a a way to import the whole conftest

Reply:
> There is a a way to import the whole conftest

Should I put the code inside `review/tests/conftest.py` in `conftest.py` instead?

Reply:
No